azconfig/client/src/main/java/com/azure/azconfig/models/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+/**
+ * This package contains classes for AzConfig Service SDK.
+ */
```

Comment:
More specificity in the package-info.java files - they all say the same thing at the moment (And they also have the wrong header format with the additional 'see License.txt....' stuff.